packages/utils/src/index.ts

```diff
@@ -105,13 +105,22 @@ export async function getTranslations(
   certificateLanguages: string[],
   schemaConfig: SchemaConfig,
 ): Promise<Translations> {
+  const errors = [];
   const translationsArray = await Promise.all(
     certificateLanguages.map(async (lang) => {
       const filePath = getRefSchemaUrl(schemaConfig, `${lang}.json`).href;
-      return { [lang]: (await loadExternalFile(filePath, 'json')) as any };
+      try {
+        return { [lang]: (await loadExternalFile(filePath, 'json')) as any };
+      } catch (error) {
+        errors.push(lang);
+      }
```

Comment:
You must return a value below `errors.push` even just a `return null`

Reply:
So I intentionally didnt return because it will throw an error anyway so there is no reason to return it. In the array on that index there will be undefined or null I dont know by hart, but that array wont be used later because of the error.

Reply:
But I do it for you.

Reply:
It's just another code quality check, based on a lint rule called consistent return, and it's also what blocked the build from passing yesterday.